packages/fx-core/tests/common/local/portChecker.test.ts

```diff
@@ -70,7 +70,7 @@ describe("portChecker", () => {
 
     it("happy path", async () => {
       const mockServer = new MockServer([]);
-      sinon.stub(net, "createServer").returns(mockServer as unknown as net.Server);
+      sinon.stub(net, "Server").returns(mockServer as unknown as net.Server);
```

Comment:
Is it possible to directly mock `detect`?

Reply:
Yes, mocked `detect-port` instead.